Added 10/02 Upcoming Releases Update (#47108)
gravitational/teleport: docs/pages/upcoming-releases.mdx

*Delayed from Teleport 16.2.0.*

#### Signed and notarized macOS assets

macOS teleport.pkg installer will include signed and notarized `tsh.app` and `tctl.app`
so downloading a separate tsh.pkg to use Touch ID will no longer be necessary.

In addition, Teleport 17 event-handler and Terraform provider for macOS will also be
signed and notarized.

#### Datadog Incident Management plugin for access requests

Teleport will support PagerDuty-like integration with Datadog's on-call and incident APIs
for access request notifications.

#### Hosted Microsoft Teams plugin for access requests

> Users will be able to setup Microsoft Teams integration for access request notifications

Reviewer: ```suggestion
Users will be able to set up Microsoft Teams integration for access request notifications
```

> using Teleport web UI without needing to self-host the plugin.

Reviewer: ```suggestion
using the Teleport web UI without needing to self-host the plugin.
```

### 17.1.0

> #### Github access

Reviewer: ```suggestion
#### GitHub access
```

Teleport will natively support Github access allowing users to transparently interact with
Github with RBAC and audit logging support.

#### Per-session MFA via IdP

Teleport users will be able to satisfy per-session MFA checks by authenticating with an external
identity provider as an alternative to using second factors registered with Teleport.

#### Database access for PostgreSQL via web UI

Database access users will be able to connect to PostgreSQL databases connected to Teleport right

> from the web UI and use psql-style interface to query the database.

Reviewer: ```suggestion
from the web UI and use a psql-style interface to query the database.
```

#### Hosted email plugin for access requests

> Users will be able to setup Mailgun or generic SMTP server for access request notifications

Reviewer: ```suggestion
Users will be able to set up Mailgun or generic SMTP server for access request notifications
```
